src/libraries/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -158,7 +158,7 @@ yield return
                 };
         }
 
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotMobile))]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotMobile), nameof(PlatformDetection.HasHostExecutable))]
```

Comment:
`SkipOnPlatform` below can be deleted. They are redundant with `PlatformDetection.IsNotMobile`